libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -310,122 +290,43 @@ - (void)enumerateDelegates:(void (^)(id<SFUserAccountManagerDelegate>))block
     }
 }
 
-#pragma mark - Temporary User
-
-- (SFUserAccount *)temporaryUser {
-    SFUserAccount *tempAccount = (self.userAccountMap)[self.temporaryUserIdentity];
-    if (tempAccount == nil) {
-        tempAccount = [self createUserAccount];
-    }
-    return tempAccount;
-}
-
 #pragma mark - Anonymous User
-
-+ (BOOL)isUserAnonymous:(SFUserAccount*)user {
-    if (nil == user.accountIdentity) {
-        return NO;
-    }
-    return [user.accountIdentity.userId isEqualToString:SFUserAccountManagerAnonymousUserAccountUserId] &&
-        [user.accountIdentity.orgId isEqualToString:SFUserAccountManagerAnonymousUserAccountOrgId];
-}
-
-- (BOOL)supportsAnonymousUser {
-    return [[[NSBundle mainBundle] objectForInfoDictionaryKey:kSFUserAccountSupportAnonymousUsage] boolValue];
-}
-
-- (BOOL)autocreateAnonymousUser {
-    return [[[NSBundle mainBundle] objectForInfoDictionaryKey:kSFUserAccountAutocreateAnonymousUser] boolValue];
-}
-
 - (BOOL)isCurrentUserAnonymous {
```

Comment:
+1.  The user hasn't logged in yet == there's no user.  I don't understand why we're keeping this.